src/main.rs

```diff
@@ -57,7 +61,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let host = IpAddr::from_str(&host_raw)?;
     let port = env::var("PORT").unwrap_or_else(|_| "80".into()).parse()?;
 
-    let client = Client::new(env::var("DISCORD_TOKEN")?);
+    let clients: ClientMap = Arc::new(DashMap::new());
```

Comment:
Does this need to be typed? Shouldn't it be inferred by the method call to `handle_request`?